.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
visual studio code settings, not sure if they're intended to be in?

Reply:
No, good point!